kiwi/builder/live.py

```diff
@@ -254,12 +254,14 @@ def create(self):
         live_container_image = FileSystem(
             name='squashfs',
             device_provider=None,
-            root_dir=self.live_container_dir
+            root_dir=self.live_container_dir,
+            custom_args={
+                'compression': self.xml_state.
+                build_type.get_squashfscompression()
+            }
         )
         container_image = NamedTemporaryFile()
-        live_container_image.create_on_file(
-            container_image.name
-        )
+        live_container_image.create_on_file(container_image.name)
```

Comment:
why was this changed ? just nitpick but the diff part has no meaning and I liked the style before better :)

Reply:
I guess this is an unintended change that escaped me. I did several changes here and there before doing the commit and I guess I did not revert to the exact same line breaks. Gonna leave it as it was.